diff -urN OpenBoard-master.orig/OpenBoard.pro OpenBoard-master/OpenBoard.pro
--- OpenBoard-master.orig/OpenBoard.pro	2023-12-08 16:30:47.000000000 +0100
+++ OpenBoard-master/OpenBoard.pro	2023-12-10 20:47:56.849928099 +0100
@@ -466,11 +466,19 @@
     LIBS += -lX11
 
     greaterThan(QT_MAJOR_VERSION, 5) {
-        LIBS += -lquazip6
-        INCLUDEPATH += "/usr/include/quazip6"
-    } else {
-        LIBS += -lquazip5
-        INCLUDEPATH += "/usr/include/quazip5"
+        #LIBS += -lquazip6
+        #INCLUDEPATH += "/usr/include/quazip6"
+ 				INCLUDEPATH += "/usr/include/Quazip-Qt6-1.4"
+				message(System Quazip)
+				CONFIG += link_pkgconfig
+				PKGCONFIG+= quazip1-qt6
+   } else {
+        #LIBS += -lquazip5
+        #INCLUDEPATH += "/usr/include/quazip5"
+				INCLUDEPATH += "/usr/include/Quazip-Qt5-1.4"
+				message(System Quazip)
+				CONFIG += link_pkgconfig
+				PKGCONFIG+= quazip1-qt5
     }
 
     LIBS += -lpoppler
